components/oauth/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -57,6 +63,35 @@ private EndpointUtil() {
     }
 
     /**
+     * Returns the {@code DefaultWebFingerProcessor} instance
+     *
+     * @return
+     */
+    public static DefaultWebFingerProcessor getWebFingerService() {
```

Comment:
Why can't we use OSGi declarative service components to get the service references?